test/libyul/yulSyntaxTests/eof/eofcreate_invalid_object_name_path.yul

```diff
@@ -0,0 +1,16 @@
+object "a" {
+    code {
+        mstore(0, eofcreate("a.b", 0, 0, 0, 0))
```

Comment:
For completeness, another distinct case that's not yet covered would be an attempt to use `a` itself.

Reply:
Added